src/utils.js

```diff
@@ -15,5 +18,27 @@ function warnOnce(domain, ...args) {
     console.warn(`${Date.now()}:`, ...args)
   }
 }
+
+const loadLedger = async () => {
+  const ledgerFile = 'https://raw.githubusercontent.com/1Hive/pollen/gh-pages/data/ledger.json'
+  const ledgerRaw = await (await fetch(ledgerFile)).text()
```

Comment:
Would be nice to handle error in case fetching fails for some reason

```suggestion
  const ledgerFileURI = 'https://raw.githubusercontent.com/1Hive/pollen/gh-pages/data/ledger.json'
  const ledgerFileResponse = await fetch(ledgerFileURI)
  if (!ledgerFileResponse.ok) {
    const message = `An error has occured: ${response.status}`;
    throw new Error(message);
  }
  const ledgerRaw = await ledgerFileResponse.text()
```